src/pages/recipes.jsx

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import { Link } from "gatsby";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+
+
+import Layout from "../components/layout";
+import SEO from "../components/seo";
+
+const Wrapper = styled.div`
+  margin: 50px 20px;
+  padding: 10px;
+`;
+const Image= styled.img`
+width: 600px;
+height: 500px;
+`
+const Recipes = ({ data }) => {
+  const page = data.allContentfulRecipes.edges;
+  return (
+    <Layout>
+      <SEO title="Recipes" />
+      {page.map(({ node }) => {
+        const title = node.title || node.slug;
+        return (
+          <Wrapper>
+            <h2 key={node.id}>
+              <Link to={`/${node.slug}`}>{title}</Link>
+            </h2>
+            <Image src={node.image.file.url} />
+            <div
+              dangerouslySetInnerHTML={{
+                __html: node.description.childMarkdownRemark.html,
+              }}
+            />
```

Comment:
This is very very very VERY bad :S

If this is a markdown file, I suggest you look for the remark plugin to embed this as parsed markdown. If it's a rich text,  check how to use the contentful rich text renderer library

Reply:
shit :(
I'm not used richtext in contentful just normal text and that what I found to solve this issue, 
gonna change it